Cargo.toml

```diff
@@ -18,6 +18,9 @@ maplit = "1.0.2"
 proptest = "1"
 tempfile = "3.3.0"
 
+[dependencies]
+candid = { version = "0.8.4", optional = true}
```

Comment:
It is implemented as a feature, so the dependency is optional.

Reply:
That's also not ideal because it complicates the development and the infrastructure.
* We must bump the candid versions when new versions appear (0.9 is coming soon, and it's a breaking change).
* We create a dependency between candid and stable structure versions that can make migrating to newer versions of `stable-structures` unnecessarily burdensome.
* Developers of `stable-structues` need to compile and test with features enabled; we must adjust the CI to test the entire feature matrix.

I don't think it's a fair bargain to win a single type conversion.